### DISCUSSION TITLE
Does `github` mode support monorepos?

### DISCUSSION THREAD

**Discussion by jmduke** [Thinkmill/keystatic#412] [Q&A] [answered]
Hi gang! Huge fan of keystatic — very excited to port over much of my structured data.

I was experimenting shifting from local to github when I discovered that the admin interface could not find any existing content. I think this is because the repo I connected keystatic to is a monorepo: the Next app that contains the keystatic integration is not at root but at `marketing/`, and when I create new items within the Keystatic UI they are committed not to `./marketing/` but at root.

Is there a good way to specify a relative path within the repository for Keystatic?

**Comment by lukebennett88** [comment, marked as answer]
You can definitely use Keystatic in a monorepo with the GitHub storage mode — you just need to set up the reader and the API route to point to the root of the repo.
You can see a working example of this on my personal website:
https://github.com/lukebennett88/v3.lukebennett.com.au
